GiniSwitchSDK/GiniSwitchSDK.podspec

```diff
@@ -8,30 +8,29 @@
 
 Pod::Spec.new do |s|
   s.name             = 'GiniSwitchSDK'
-  s.version          = '0.1.0'
-  s.summary          = 'A short description of GiniSwitchSDK.'
-
-# This description is used to generate tags and improve search results.
-#   * Think: What does it do? Why did you write it? What is the focus?
-#   * Try to keep it short, snappy and to the point.
-#   * Write the description between the DESC delimiters below.
-#   * Finally, don't worry about the indent, CocoaPods strips it!
+  s.version          = '0.4.0'
```

Comment:
I'll have 0.4 while I'm testing the Cocoapods release process. Then we can decide what to go to.